RELEASE_NOTES.md

```diff
@@ -18,7 +18,28 @@ CSM 1.2 contains approximately 2000 changes spanning bug fixes, new feature deve
 * Release Cray Automated Network Utility - CANU V1.0.0
 * Performance improvements to Unbound and DHCP Helper
 * Initial Release of Bifurcated CAN
-* PowerDNS including PowerDNS Manager and Troubleshooting Guide
+
+  The user and administrative traffic segregation introduced by Bifurcated CAN has changed the URLs for certain services as it is now necessary to include the network path in the fully qualified domain name.
```

Comment:
May want to be specific that only the API service is available on all both CMN and CAN/CHN.   The rest are CMN-only.   It should be obvious from the examples, but just to be crystal clear.

Reply:
Added language to that effect.